tox.ini

```
@@ -102,6 +102,8 @@ changedir = {toxinidir}/docs
 isolated_build = true
 passenv =
   SSH_AUTH_SOCK
+  READTHEDOCS
+  READTHEDOCS_VERSION_TYPE
```

Let's order these alphabetically:
```suggestion
  READTHEDOCS
  READTHEDOCS_VERSION_TYPE
  SSH_AUTH_SOCK
```

---

I'll amend the commit with this.

---

Ack. Marking as resolved.